node/src/transport.rs

```diff
@@ -21,20 +21,25 @@ mod imp {
         keypair: &Keypair,
     ) -> Result<Boxed<(PeerId, StreamMuxerBox)>, P2pError> {
         let quic_transport = {
-            let mut config = quic::Config::new(keypair);
-
-            // Celestia uses draft 29
-            config.support_draft_29 = true;
-
+            let config = quic::Config::new(keypair);
             quic::tokio::Transport::new(config)
         };
 
         let tcp_transport = {
             let mut yamux_config = yamux::Config::default();
 
-            // This increases throughput
-            yamux_config.set_receive_window_size(2 * 1024 * 1024);
-            yamux_config.set_max_buffer_size(3 * 1024 * 1024);
+            // This increases bandwidth utilitation. With 1MB of receive
+            // window, we can utilize 81.92mbps (10mb/s) per stream when
+            // latency is 100ms: 1mb / 100ms * 8bits = 81.92mbps
+            //
+            // This means that machine needs 8gb of ram to handle 8192
+            // streams (the default). For this reason we lower max streams
+            // to 2048.
+            //
+            // More info: https://github.com/libp2p/rust-yamux/issues/162
```

Comment:
wondering about also an info that Go impl has the max window as 16MB and dropping a link to
https://github.com/libp2p/go-libp2p/blob/master/p2p/muxer/yamux/transport.go#L24
leaving this up to you

Reply:
done